Samples/SensorActivityMonitorConsoleApp/readme.md

```diff
@@ -0,0 +1 @@
+SensorActivityMonitorConsoleApp is the full compileable project seen in the [IMFSensorActivityMonitor API documentation in docs.microsoft.com](https://docs.microsoft.com/windows/win32/api/mfidl/nn-mfidl-imfsensoractivitymonitor)
```

Comment:
typo: compilable

Reply:
lets just remove the word - it is assumed that the github sample can be fully built compiled and linked.